projects/igniteui-angular/src/lib/grids/summaries/grid-summary.service.ts

```diff
@@ -121,7 +121,7 @@ export class IgxGridSummaryService {
         }
         this.grid.columns.filter(col => col.hasSummary).forEach((column) => {
             if (!rowSummaries.get(column.field)) {
-                const summaryResult = column.summaries.operate(data.map(r => resolveNestedPath(r, column.field)),
+                const summaryResult = column.summaries.operate(data.map(r => resolveNestedPath(r, column.field), groupRecord),
```

Comment:
in grid-smmary,ts, groupRecord is the 4th argument, while here it is sent as second parameter.